requirements/customer/scheduling-algorithm.tex

```diff
@@ -0,0 +1,66 @@
+\subsection{Scheduling Algorithm}
```

Comment:
Why is this file being added in this PR.

Reply:
I guess @M1keReck merged master instead of rebasing on top of it.

Reply:
I did run into some issues when I added the most recent changes, so I might have messed some things up.